docs/nlu/components.rst

```diff
@@ -456,6 +441,38 @@ EmbeddingIntentClassifier
               starspace algorithm in the case ``mu_neg = mu_pos`` and ``use_max_sim_neg = False``.
               See `starspace paper <https://arxiv.org/abs/1709.03856>`_ for details.
 
+
+.. _keyword_intent_classifier:
+
+KeywordIntentClassifier
+~~~~~~~~~~~~~~~~~~~~~~~
+
+:Short: Simple keyword matching intent classifier, intended for small, short-term projects.
+:Outputs: ``intent``
+:Requires: nothing
+:Output-Example:
+
+    .. code-block:: json
+
+        {
+            "intent": {"name": "greet", "confidence": 1.0}
+        }
+
+:Description:
+    This classifier works by searching a message for keywords.
+    The matching is case sensitive by default and searches only for exact matches of the keyword-string in the user message.
+    The keywords for an intent are the examples of that intent in the NLU training data.
+    This means the entire example is the keyword, not the individual words in the example.
+    .. note:: This classifier is intended only for small projects or to get started. If you have few training data you can use the ``pretrained_embeddings_spacy`` pipeline to get started.
```

Comment:
```suggestion
    .. note:: This classifier is intended only for small projects or to get started. If you have NLU training data you can use one of our pipelines :ref:`choosing-a-pipeline`.
```

Reply:
Please break long lines

Reply:
I thought lines are done automatically, I'll fix it

Reply:
not for docs